manticore/core/workspace.py

```diff
@@ -415,6 +423,23 @@ def __init__(self, desc=None):
         self._id_gen = manager.Value('i', self._last_id)
         self._lock = manager.Condition(manager.RLock())
 
+    def testcase(self, prefix='test'):
```

Comment:
not for now, but TODO: this feels kind of structure feels awkward. the increasing prominence of APIs here and there that are used by only EVM stuff or only Linux is a sign there is deep restructuring we need to do.

Reply:
Yes!!! 
Please have only one of this testcase() in the restructuring !